products/spring-boot.md

```diff
@@ -19,6 +19,8 @@ auto:
 
 identifiers:
 -   purl: pkg:maven/org.springframework.boot/spring-boot
+-   purl: pkg:maven/spring-boot/spring-boot
```

Comment:
There is nothing at https://central.sonatype.com/artifact/spring-boot/spring-boot.